tests/utils/esx/vdvs.go

```diff
@@ -15,7 +15,7 @@
 // This util provides various helper methods that can be used by different tests to
 // fetch information related to vSphere docker-volume-service.
 
-package verification
+package esx
```

Comment:
This should be utis/docker, its checking for a docker managed plugin.

Should also be checking that docker is running